hbase-oss/src/main/java/org/apache/hadoop/hbase/oss/sync/ZKTreeLockManager.java

```diff
@@ -196,7 +197,7 @@ protected void readUnlock(Path p) throws IOException {
   We need to protect this block against potential concurrent calls to close()
    */
   @Override
-  protected synchronized boolean writeLockAbove(Path p) throws IOException {
+  protected boolean writeLockAbove(Path p) throws IOException {
```

Comment:
HBASE-24961 #14 introduced the synchronized here. I think Wellington was fixing an issue where SecureBulkLoadManager was causing problems with the `FileSystem.CACHE`. That one was caused by HBASE-23679 which I had tried to fix. Need to make sure we have testing for bulk loading to make sure we don't regress.